src/cluster/dcommands/general/define.ts

```diff
@@ -74,7 +74,7 @@ const wordApiMapping = mapping.object({
         definition: mapping.string,
         partOfSpeech: mapping.string,
         synonyms: mapping.array(mapping.string).optional
-    })),
+    }, { strict: false })),
```

Comment:
What does it mean?

Reply:
object mappings are strict by default which means that the source object cannot have any excess properties that you dont map. Making it not strict means that it only requires the properties you do define